jobs/build/build-sync/Jenkinsfile

```diff
@@ -68,18 +68,20 @@ node {
     }
 
     try {
-	// This stage is safe to run concurrently. Each build runs
-	// these steps in its own directory.
-        stage("Generate inputs") { build.buildSyncGenInputs() }
-	// Allow this job to run concurrently for different
-	// versions. That is to say, do not allow builds for the same
-	// version to run the business logic concurrently.
-        lock("mirroring-lock-OCP-${params.BUILD_VERSION}") {
-            stage("oc image mirror") { build.buildSyncMirrorImages() }
-	}
-        lock("oc-applying-lock-OCP-${params.BUILD_VERSION}") {
-            stage("oc apply") { build.buildSyncApplyImageStreams() }
-        }
+		sshagent(["openshift-bot"]) {
```

Comment:
hm. the indentation may have been wonky in this file before you started. definitely don't like using tabs, they mess up looking at diffs among other things.
could we use basically the same standard as python? 4-space indents (no tabs)?